src/party_context_menu.c

```diff
@@ -431,7 +431,7 @@ void PartyMenu_RemoveAllWindows(PartyMenuStruct *partyMenu) {
     }
 }
 
-void PartyMenu_SetContextMenuStaticStrings(PartyMenuStruct *partyMenu) {
+void PartyMenu_SetContextMenuStaticStrings(PartyMenu *partyMenu) {
     ReadMsgDataIntoString(partyMenu->msgData, msg_0300_00128, partyMenu->contextMenuStrings[PARTY_MON_CONTEXT_MENU_SWITCH]);
```

Comment:
I still think it's a good idea to comment what the messages are but I can do that in a separate PR later if you don't wanna do it rn